src/ucs/debug/debug.c

```diff
@@ -1150,7 +1143,8 @@ sighandler_t signal(int signum, sighandler_t handler)
     return orig(signum, handler);
 }
 
-int sigaction(int signum, const struct sigaction *act, struct sigaction *oact)
+int ucs_debug_sigaction(int signum, const struct sigaction *act,
```

Comment:
the logic of calling original handler should be part of libucs_signal.so
libucs_signal.so should call libucs function, to understand which parameters to pass to the original handler.
for example:
```c
int ucs_debug_is_signal_handler_forced(int signum) 
{
   return ucs_debug_initialized && ucs_debug_is_error_signal(signum);
}
```

and in libucs_signal.so, use this function result to determine behaior

Reply:
> the logic of calling original handler should be part of libucs_signal.so

ok, moved some evaluations into UCS_signal